File: resourcespoet/src/test/java/com/commit451/resourcespoet/CommentTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

/**
 * Tests the resources creation
 */
public class CommentTest {

    @Test
    public void commentTest() throws Exception {
        ResourcesPoet poet = ResourcesPoet.create()
                .addComment("This is a comment");

        TestUtil.assertEquals("comment.xml", poet);
    }
}

File: resourcespoet/src/test/java/com/commit451/resourcespoet/CommentTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class CommentTest {

    @Test
    fun commentTest() {
        val poet = ResourcesPoet.create()
                .addComment("This is a comment")

        TestUtil.assertEquals("comment.xml", poet)
    }
}
